### PR TITLE
fix: Support 3 types of syntax for OpenID Connect Issuer Host

### PR DESCRIPTION
**Pull request by mudana-grune**
Bug: Authentication settings with OpenID Connect fails #6030
https://youtrack.weseek.co.jp/issue/GW-7821

fix https://github.com/weseek/growi/issues/6030

# Testing Steps
## Test issuerHost with hostname only
hostname
### Result (Success)
image

## Test with host URL 
image
### Result (success)
image

## Test with full configuration path 
image
### Result (Success)
image

## Test with host URL (different url format, config test only)
image
### Result (Success)
image

## Test with full configuration path  (different url format, config test only)
image
### Result (Success)
image

### CODE REVIEW

File: packages/app/src/server/service/passport.ts [hunk 1]
```diff
@@ -719,16 +719,26 @@ class PassportService implements S2sMessageHandlable {
...
+   * Acceptable formats :
+   * - https://hostname.com/auth/
+   * - domain only (hostname.com)
+   * - Full metadata url (https://hostname.com/auth/v2/.well-known/openid-configuration)
    * @param issuerHost string
-   * @returns string URL.origin
+   * @returns string URL/.well-known/openid-configuration
    */
-  getOIDCIssuerHostName(issuerHost) {
+  getOIDCMetadataURL(issuerHost) {
```
Comment (hakumizuki) [inline packages/app/src/server/service/passport.ts:729]: Please specify the args type as well.

Comment (hakumizuki) [inline packages/app/src/server/service/passport.ts:729]: Please specify the args type as well.

File: packages/app/src/server/service/passport.ts [hunk 2]
```diff
@@ -741,12 +751,12 @@ class PassportService implements S2sMessageHandlable {
  */
   async isOidcHostReachable(issuerHost) {
```
Comment (hakumizuki) [inline packages/app/src/server/service/passport.ts:752]: Please specify the args type as well.

**Review by hakumizuki — CHANGES_REQUESTED** [review]
Could you also fix the typo of `getOIDCIssuerInstace` method as well?

File: packages/app/src/server/service/passport.ts (hunk 1 above)
Comment (mudana-grune) [reply packages/app/src/server/service/passport.ts]: - Rename `getOIDCIssuerInstace`  to `getOIDCIssuerInstance` 
- Add params and retuen type

File: packages/app/src/server/service/passport.ts (hunk 2 above)
Comment (mudana-grune) [reply packages/app/src/server/service/passport.ts]: - Add params and return type

**Review by hakumizuki — APPROVED** [review]

**Pull request merged**